agent/exploits/cve_2014_0780.py

```diff
@@ -18,7 +18,6 @@
 
 DEFAULT_TIMEOUT = 90
```

Comment:
use datetime.timedelta(seconds=90)

Reply:
outside of the scope my friend, there is ~70 exploit class. fixing code is a different task.
this is to use cloudscraper instead of requests.